src/commands/functions/functions-create.js

```diff
@@ -189,6 +198,42 @@ const pickTemplate = async function ({ language: languageFromFlag }) {
 
 const DEFAULT_PRIORITY = 999
 
+const selectTypeOfFunc = async () => {
+  const { functionType } = await inquirer.prompt([
+    {
+      name: 'functionType',
+      message: "Select the type of function you'd like to create: ",
+      type: 'list',
+      choices: ['Edge Function (Deno)', 'Serverless Function (Node/Go)'],
+    },
+  ])
+  return functionType
+}
+
+const ensureEdgeFuncDirExists = function (command) {
+  const siteId = command.netlify.site.id
+
+  if (!siteId) {
+    error(`${NETLIFYDEVERR} No site id found, please run inside a site directory o \`netlify link\``)
+  }
+
+  const functionsDirHolder = 'netlify/edge-functions'
```

Comment:
While `netlify/edge-functions` is the default directory for edge functions, it's possible for people to specify their own. If they do, we should respect it and not mandate our own.

Reply:
Oh interesting. I know you can specify a directory for regular serverless functions in the UI or in the netlify.toml file, and was wondering about this, but couldn't find anything about setting a custom ef directory in the docs. Is this something only possible in the toml file? Should this be added to the docs somewhere?

Reply:
> Should this be added to the docs somewhere?

I'm [tracking this as something to document as part of GA readiness](https://www.notion.so/netlify/Save-for-GA-89e6e2871d09447cbc5da15c18e36605#66d30c114db946319cc03d93e3351aca) but am unlikely to add it to the docs before then

Thanks for thinking about docs @jenae-janzen !!

Reply:
@eduardoboucas, do you have any more information about how a user can set a custom edge function directory? I tried a few different versions of this in my toml file but no dice:
<img width="207" alt="Screen Shot 2022-06-28 at 1 54 58 PM" src="https://user-images.githubusercontent.com/101715009/176250611-45c24096-0b10-4f3f-8f47-a0e4d2d64e34.png">
<img width="610" alt="Screen Shot 2022-06-28 at 1 59 06 PM" src="https://user-images.githubusercontent.com/101715009/176251011-bbef0068-a2f3-4022-b596-4eee40562da1.png">

Reply:
Oops, sorry for the late response. It should be:

```toml
[build]
edge_functions = "some-directory"
```

This is indeed a flow that we haven't documented and that we need to spend some time testing before GA. In any case, I think it would be good for you to retrieve the path from `config.edge_functions` (in the same way we're doing for serverless functions) as opposed to just hardcoding the default value.

Reply:
Good to know! I added a check to see if they have it configured and to use their custom directory if they do. Right now, I'm still using `netlify/edge-functions` as the default if they don't, but could also prompt them to choose a directory for their edge functions to live in and append their choice to the `toml`.

To me, it feels a bit safer to just use `netlify/edge-functions` so people aren't confused running into an undocumented workflow, but what do you think?